TestModels/SharedMakefile.mk

```diff
@@ -89,6 +93,10 @@ dafny-reportgenerator:
 
 # Dafny helper targets
 
+# Transpile the entire project's impl
+_transpile_implementation_all: TRANSPILE_DEPENDENCIES=$(patsubst %, -library:$(PROJECT_ROOT)/%, $(PROJECT_INDEX))
+_transpile_implementation_all: transpile_implementation
```

Comment:
Java and NET call this target.
It looks like we can consolidate this into `transpile_implementation`, but (as you mentioned) for the sake of SharedMakefile consolidation it might be good to leave as is.